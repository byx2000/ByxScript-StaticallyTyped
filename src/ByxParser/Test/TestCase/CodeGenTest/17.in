int g_i = 100;
double g_x = 3.14;

function void main()
{
	
}

function int f()
{
	return g_i;
}

function double g()
{
	return g_x;
}