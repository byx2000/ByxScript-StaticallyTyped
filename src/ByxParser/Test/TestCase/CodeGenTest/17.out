3
0 0
0 5
0 8

2

iconst 100
igstore 0
dconst 3.14
dgstore 1
ret

igload 0
ret
ret

dgload 1
ret
ret